cmd/go.mod

```diff
@@ -36,4 +36,8 @@ replace (
 
 	// Temporary fork for avoiding importing patent-protected code: https://github.com/hashicorp/golang-lru/issues/73
 	github.com/hashicorp/golang-lru => github.com/ktock/golang-lru v0.5.5-0.20211029085301-ec551be6f75c
+
+	// Ensure v1.3.1+ is used to address https://github.com/advisories/GHSA-g3vv-g2j5-45f2
+	// TODO: remove this replace dirctive once all dependencies are updated to point to v1.3.1+
+	github.com/ipld/go-codec-dagpb => github.com/ipld/go-codec-dagpb v1.3.2
```

Comment:
Can't you just pin this version as an `// indirect` dependency in the `require()` section?
Maybe with `go 1.17` format.

Reply:
`github.com/ipld/go-codec-dagpb v1.3.2` is added to `go.sum` but it still contains `github.com/ipld/go-codec-dagpb v1.3.0` entry (using go 1.17 didn't help).
I'm not sure this solves the dependabot alert.
Can we merge this and check if the dependabot alert is solved? If not solved, we'll need to use `replace`.